frontend/src/pages/secret-manager/OverviewPage/components/SecretOverviewTableRow/SecretOverviewTableRow.tsx

```diff
@@ -258,34 +255,43 @@ export const SecretOverviewTableRow = ({
-                              secretId={secret?.id}
-                              isOverride={Boolean(secret?.idOverride)}
-                              isImportedSecret={isImportedSecret}
-                              importedSecret={importedSecret}
-                              isCreatable={isCreatable}
-                              onSecretDelete={onSecretDelete}
-                              onSecretCreate={onSecretCreate}
-                              onSecretUpdate={onSecretUpdate}
-                              environment={slug}
-                              isRotatedSecret={secret?.isRotatedSecret}
-                              importedBy={importedBy}
-                              isSecretPresent={Boolean(secret)}
-                            />
+                          <td className="col-span-2 w-full">
+                            <div className="group flex flex-col gap-0.5 divide-y divide-mineshaft-600">
+                              <div className="flex h-8 items-center">
+                                <SecretEditRow
+                                  secretPath={secretPath}
+                                  isVisible={isSecretVisible}
+                                  secretName={secretKey}
+                                  isEmpty={secret?.isEmpty}
+                                  secretValueHidden={secret?.secretValueHidden || false}
+                                  defaultValue={getDefaultValue(secret, importedSecret)}
+                                  secretId={secret?.id}
+                                  hasOverride={Boolean(secret?.idOverride)}
+                                  isImportedSecret={isImportedSecret}
+                                  importedSecret={importedSecret}
+                                  isCreatable={isCreatable}
+                                  onSecretDelete={onSecretDelete}
+                                  onSecretCreate={onSecretCreate}
+                                  onSecretUpdate={onSecretUpdate}
+                                  environment={slug}
+                                  isRotatedSecret={secret?.isRotatedSecret}
+                                  importedBy={importedBy}
+                                  isSecretPresent={Boolean(secret)}
+                                />
+                              </div>
+                              {Boolean(secret?.idOverride) && (
+                                <SecretPersonalOverrideView
+                                  isVisible={isSecretVisible}
+                                  secretKey={secretKey}
+                                  environment={secret?.env || ""}
+                                  secretPath={secretPath}
+                                  projectId={currentProject.id}
+                                  className="w-full space-x-2"
+                                />
+                              )}
+                            </div>
```

Comment:
The block that renders both the original secret and the override on the overview page